main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -1,56 +1,91 @@
+the chain where the denom is held, and the chain that issues the corresponding asset. 
+See [getBrandInfo](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#getBrandInfo).
 
 ```javascript
-const stakeAndSwapFn = async (orch, ...) => {
-...
-  const omni = await orch.getChain('omniflixhub');
-  const agoric = await orch.getChain('agoric');
+const brandInfo = orchestrator.getBrandInfo('denom');
+```
 
-  const [omniAccount, localAccount] = await Promise.all([
-    omni.makeAccount(),
-    agoric.makeAccount(),
-  ]);
+- `asAmount`converts a denom amount to an `Amount` with a brand. See [asAmount](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#asAmount).
 
-  const omniAddress = omniAccount.getAddress();
+```javascript
+const amount = orchestrator.asAmount({ denom: 'uatom', value: 1000n });
+```
 
-  // deposit funds from user seat to LocalChainAccount
-...
-  const transferMsg = orcUtils.makeOsmosisSwap({ ... });
+## Orchestration Account
 
-  try {
-    await localAccount.transferSteps(give.Stable, transferMsg);
-    await omniAccount.delegate(offerArgs.validator, offerArgs.staked);
-  } catch (e) {
-    console.error(e);
-  }
-};
+Orchestration accounts are a key concept in the Agoric Orchestration API, represented by the 
+[`OrchestrationAccountI`](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.OrchestrationAccountI) interface. These accounts provide high-level operations for managing 
+accounts on remote chains, allowing seamless interaction and management of interchain
+accounts. The orchestration accounts abstract the complexity of interchain interactions,
+providing a unified and simplified interface for developers.
+
+### 1. Address Management
+
+- `getAddress` retrieves the address of the account on the remote chain.
+
+```javascript
+const address = await orchestrationAccount.getAddress();
+```
+
+### 2. Balance Management
+
+- `getBalances` returns an array of amounts for every balance in the account.
+- `getBalance` retrieves the balance of a specific denom for the account.
+
+```javascript
+const balances = await orchestrationAccount.getBalances();
+const balance = await orchestrationAccount.getBalance('uatom');
+```
+
+### 3. Funds Transfer
+
+- `send` transfers an amount to another account on the same chain.
+- `transfer` transfers an amount to another account, typically on another chain.
+- `transferSteps` transfers an amount in multiple steps, handling complex transfer paths.
+- `deposit` deposits payment from Zoe to the account. For remote accounts, an IBC Transfer
+will be executed to transfer funds there.
+
+```javascript
+await orchestrationAccount.send(receiverAddress, amount);
+await orchestrationAccount.transfer(amount, destinationAddress);
+await orchestrationAccount.transferSteps(amount, transferMsg);
+await orchestrationAccount.deposit(payment);
 ```
 
-### ChainHub
+## ChainHub
```

Comment:
Please make this section (ChainHub) consistent with previous two sections with regard to listing different APIs separately. For example I think you can add two sub-headings for registerChain and registerConnection in this section.

Reply:
i agree, and take the opportunity to explain each. This is a good set of api functions though

Reply:
@amessbee  i'd say its good this is prob the only comment to address before merging.